laminar-markets/aptos-core
# syncfork

- Add example of Resource Account implementation from Discord discussion
- Add more details from Discord
- Add Petra Wallet and Aptos resources updates to What's New in Docs
- Fully qualify Have fun link
- Add Explorer docs to What's new in Docs
- Revising explorer docs ref per feedback
- [docs] fix whats new in docs
- [backup] Fix toml -> yaml for backup verify (#5089)
- [testnet] Add ability to pass storage class to testnet (#5031)
- [tf/gcp][fullnode] ensure namespace and k8s serviceaccount name for b… (#5099)
- [mempool] add bucket to latency stats (#5057)
- Update getting-started.md
- Update getting-started.md
- Update getting-started.md
- [Docs] PFN install doc update and a deployment table fix.
- [docs] Fix node docker docs
- [docs] add doc to use fullnode from binary instead of with cargo run
- [mempool] smoke test to confirm failover on default_failovers=0 (#4953)
- [framework] always check compat unless arbitrary policy
- [move] Use workspace deps for Move and upgrade to Rust 1.64 (#5113)
- [token] added functions for mutating token data (#5017)
- [storage] consolidate save_transactions (#5088)
- roll back pnpm default package manager
- Update base-gas.md
- [docs] Fix module name casing
- [docs] Update Account address docs
- [docs] Make 16 byte -> 32 byte addresses
- Update telemetry.md
- [logging] make sure flush doesn't block (#5126)
- [terraform][testnet] disable logger (#5132)
- [helm] consensus alerts and mainnet urls (#5103)
- [cli] Fix network argument in aptos init
- [dashboards] sync grafana dashboards (#5085)
- [gha] update base image in compat test (#5140)
- [State Sync] Improve the default config values.
- [Token] update the standard doc (#5138)
- [api] Handle unparsable utf-8 on output
- refactor fetcher code to be able to skip over bad txns (#5143)
- Fix wait for error message (#5161)
- test STORAGE_WRITE_LIMIT_REACHED
- [Docs] Fixing links and line editing. (#5164)
- [build] switch to lld on linux (#4910)
- [Docs] Calling out that Docker is not supported yet for the indexer fullnode.
- [docs] Update to use profiles everywhere in operator docs
- Start edits to Your First Transaction tutorial
- Fix caps in TypeScript
- Update output description to match latest numbers
- Update example output
- Continue edits
- Finishing edits to page
- Fix links to SDKs
- Update first-transaction.md
- Update first-transaction.md
- Update using-azure.md
- [State Sync] Small tweaks to configs.
- [vm] Transaction cleanup: create session after abort info extraction
- [API] Remove erroneous log message about reverse proxy on startup (#5185)
- [move] Enable documentation generation (#5187)
- standardize address and table handle (#5170)
- Reorder tutorials in index to match left nav (#5163)
- [smoke tests] keep logs even on flaky tests (#5167)
- fix that replay-verify replay start version is off by 1
- [backup] fix that unauthorized cloud access results in flushing metadata cache
- [gha] nightly replay-verify: in parallel
- move to an action for code sharing
- separate replay runs for testnet and mainnet
- [move] Update commit hash (#5204)
- Replace all variable-dependent links in aptos.dev with relative links (#5191)
- [move-prover] Refactor prover integration (#5208)
- [docs] Update styles & footer links to Aptos Foundation
- [tf/helm] various cleanups (#5202)
- [gha] use beefy machine to replay-verify
- [backup] fix that JmtRestore::drop() can deadlock if error occurs while channel is empty
- [State Sync] Default to execution mode.
- [consensus][logs] Demote non-critical error logs in consensus (#5186)
- [state-sync][logs] Demote non-critical error logs in StateSync (#5188)
- [network][logs] Demote non-critical error logs in Network (#5189)
- [misc][logging] Demote non-critical error logs in rest (#5190)
- [helm][aptos-node] tune probes (#5245)
- [API] Make enabled: false actually disable the API (#5244)
- [artifacts] Add a test which ensures generated Move artifacts are up-to-date (#5226)
- [tf/aws][fullnode] public-read acl for public backups (#5252)
- replay-verify jobs: make it daily
- [dashboards] sync grafana dashboards (#5141)
- bump memory and cpu for backup-verify job
- [TS][SDK] Bump timeouts on tests (#4933)
- [backup] push metrics to vmagent
- try setting kubernetes_pod_name
- [CLI] Use jemalloc for the CLI (#5249)
- [State Sync] Make num stream timeouts configurable.
- [State Sync] Reduce max message size and increase timeout.
- [State Sync] Add smoke tests to check large limits.
- [move] Updating move commit, among others fixing a bug in docgen (#5270)
- [smoke tests] test_genesis_transaction_flow: add wait checks on validator restarts (#5262)
- Ed25519 test only natives (#5272)
- [Docs] Adding rate limits section.
- [Indexer] Fix timestamp bug and restart from gap (#5214)
- [CLI] Only depend on jemalloc crate on unix (#5293)
- ed25519 test-only natives implementation tweak (#5289)
- [move] Fix artifact generation and test for pure Move development (#5297)
- Update CODEOWNERS
- Test and edit Your First NFT tutorial (#5097)
- [helm][aptos-node] remove haproxy probes (#5306)
- Split and move Install CLI docs to Getting Started section of left nav (#5098)
- Test and edit Your First Move Module (#5104)
- [dashboards] sync grafana dashboards (#5267)
- [tf] update requested and limit mem to be the same otherwise pod can be killed because of increased use of fs cache
- [smoke tests] fix flaky node panic due to rebuild in parallel tests (#5308)
- Fix broken BCS links
- [python-sdk] fix client.py offer_token and claim_token
- Edit Upgrading Move Code
- Replace outdated Move auto-gen docs with the latest and announce in What's new
- Add Aptos Move Examples directory
- chore: fix typo in aptos cli for profile creation
- Add clay-aptos to OWNERS for developer-docs-site directory
- Add token.move link to top of Aptos Token
- Update aptos-token.md
- Return error if provided config was missing or invalid
- Fix comment
- [cli] Fix move download error
- [cli] Fix lint error
- [Cargo] Add comment about performance profile.
- [tf/gcp] update default instance to recommended n2 (#5312)
- [aptos-node] better error message for the feature flag check (#5330)
- [Token] added functions to rotate collectiondata fields (#5265)
- build

### Description

### Test Plan
